static/src/javascripts/projects/common/modules/discussion/comment-count.spec.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import mediator from 'lib/mediator';
+import fastdom from 'lib/fastdom-promise';
+import {
+    init,
+    getContentIds,
+    getElementsIndexedById,
+} from 'common/modules/discussion/comment-count';
+
+const setHTML = (h: string): Promise<void> =>
+    fastdom.write(() => {
```

Comment:
Since jest is based on jsdom, I'd rather avoid adding complexity with fastdom and ditch it.